tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,93 @@
+"""Module2/task1: Includes task of creating sandwich calculator based on user's input."""
+import pyinputplus as pyip
+
+
+# Vars
```

Comment:
Such comment is unnecessary (and actually "Consts" would be better choice ;))

Reply:
Wanted to divide it into sections, but yeah, kinda not useful.